tests/Execution/long-shifts.java

```diff
@@ -0,0 +1,44 @@
+// RUN: javac %s -d %t
+// RUN: jllvm -Xenable-test-utils %t/Test.class | FileCheck %s
+
+class Test
+{
+    public static native void print(long i);
+
+    public static void main(String[] args)
+    {
+        long x = 5l;
+        long y = 2l;
+        long z = -1l;
+        long min = Long.MIN_VALUE;
+        long max = Long.MAX_VALUE;
+
+        //CHECK: 20
```

Comment:
Shouldn't these be larger? Some probably match since these are prefixes for the real results but
`51 << 2` is `202`, `21 << 5` is `672` etc.

Reply:
I apparently can't read. These are `l` not 1s